x-pack/plugins/observability/public/plugin.ts

```diff
@@ -64,7 +64,6 @@ export class Plugin implements PluginClass<ObservabilityPluginSetup, Observabili
         icon: 'logoObservability',
         path: '/app/observability',
         order: 200,
-        className: 'homSolutionsSection__magentaCard',
       });
```

Comment:
See comment above regarding why we likely need to keep this (or something similar to it).

Reply:
See my comment above as well :)